xdesign/plot.py

```diff
@@ -367,26 +377,54 @@ def discrete_phantom(phantom, size, ratio=9, uniform=True,
     image = 0
 
     if phantom.geometry is not None and phantom.material is not None \
-       and hasattr(phantom.material, prop):
+       and np.alltrue([hasattr(phantom.material, temp) for temp in prop]):
 
-        psize = 1.0 / size
+        # select whether use a fixed pixel size (at 1) or fixed object size (1cm)
+        if fix_psize:
+            psize = 1
+        else:
+            psize = 1.0 / size
```

Comment:
I don't understand the purpose of the new `fix_psize` parameter. Isn't it equivalent to setting the `size` parameter to `1`? We could also just change the interface to set `psize` directly instead of using `size`. `size` is just a holdover from when I initially designed this function for the `plot` module. It was more convenient to say "I want a 256x256 grid" instead of "I want the pixel size to be 1/256 cm".